central/probeupload/manager/manager_impl.go

```diff
@@ -5,33 +5,32 @@ import (
 	"encoding/binary"
```

Comment:
One interesting thing about the probe upload is that there is no way to remove the uploaded bundle. I understand that we may need more than one versions of modules there but we should at least allow customer to remove them.